docs/configmaps.md

```diff
@@ -0,0 +1,15 @@
+# ConfigMaps
```

Comment:
This is specifically for RHHI right?  I would call this doc `docs/kni.md`

Reply:
sure, done.